code/codegen/build.gradle.kts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2020 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    // Apply the Kotlin JVM plugin to add support for Kotlin.
+    kotlin("jvm")
+
+    id("default-config")
+}
+
+repositories {
+    maven { url = uri("https://repo.gradle.org/gradle/libs-releases-local/") }
+}
+
+tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile>().configureEach {
+    kotlinOptions {
+        jvmTarget = "1.8"
+        freeCompilerArgs = listOf("-Xjvm-default=enable")
+    }
+}
+
+kotlin {
+    sourceSets.all {
+        languageSettings.useExperimentalAnnotation("kotlin.ExperimentalStdlibApi")
+    }
+}
+
+dependencies {
+    implementation(project(":model"))
+
+    implementation(platform("org.jetbrains.kotlin:kotlin-bom"))
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
```

Comment:
you actually don't need to add a dep on the stdlib anymore.
https://kotlinlang.org/docs/reference/using-gradle.html#dependency-on-the-standard-library
(we'll need to clean up the other subprojects)

Reply:
done.